spec/job/adapters/linux_host/launcher_spec.rb

```diff
@@ -160,6 +194,30 @@ def content
             }
         ] }
 
+        let(:tmux_broken_output_with_many_returned_fields) {
+            <<~HEREDOC
+            ----------------  ------------------------------------
+            Usage Statistics for project PZS0714
+            Time              2021-09-12 to 2021-09-13
+            PI                alanc@osc.edu
+            Remaining Budget  -0.25
+            ----------------  ------------------------------------
+
+            User           Jobs    Dollars    Status
+            -------------  ------  ---------  --------
+            alanc          0       0.0        ACTIVE
+            johrstrom      0       0.0        ACTIVE
+            johrstromtest  0       0.0        ACTIVE
+            travert        0       0.0        ACTIVE
+            --             --      --
+            TOTAL          0       0.0
+            launched-by-ondemand-a8e85cd4-791d-49fa-8be1-5bd5c1009d70,1569609529,175138
+            launched-by-ondemand-a8e85cd4-791d-49fa-8be1-5bd5c1009d70,1569609529
+            launched-by-ondemand-a8e85cd4-791d-49fa-8be1-5bd5c1009d70
+            a8e85cd4-791d-49fa-8be1-5bd5c1009d70,1569609529,175138
```

Comment:
Be sure to use `#{sep}` here so that when the separator changes we don't need to update these test cases.